pywarp10/sanitize.py

```diff
@@ -51,9 +52,21 @@ def sanitize(x: Any) -> str:
             date = date.replace(tzinfo=None)
             x = date.isoformat(timespec="microseconds") + "Z"
         return f"'{x}'"
-    elif isinstance(x, bool):
+    if isinstance(x, bool):
         return str(x).upper()
-    elif isinstance(x, Iterable):
+    if isinstance(x, datetime.date):
```

Comment:
if you use pandas dates are converted to pd.datetime, should we add the case?

Reply:
pd.datetime are datetime.datetime internally

```
>>> import pandas as pd
>>> import datetime
>>> isinstance(pd.to_datetime("today"), datetime.datetime)
True
```